src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
+  root: {
+    width: "1100px",
+    margin: "0 auto",
+    background: "white",
```

Comment:
Why is the background being set to white? Should just follow the theme so its consistent with the rest of the UI and its easy to update the look globally. Similarly, we shouldn't be setting all the text to black.